hack/make/script-tools.sh

```diff
@@ -0,0 +1,12 @@
+function green {
```

Comment:
These functions are now in two places.. here and in the test.sh script.  Is there some way can share them instead of duplicating them?

Reply:
Yup!